frontend/src/app/survey/detail/response/response.component.html

```diff
@@ -31,6 +31,7 @@
             <mat-icon>add</mat-icon>
         </button>
     </div>
+    <mat-error *ngIf="endDateBeforeStartDateError">{{ " survey.invalid_date" | translate }}</mat-error>
```

Comment:
That space is probably not supposed to be there and will probably break the key lookup

Reply:
And the pipeline fails always 🤷